data/transport missions.txt

```diff
@@ -1246,3 +1246,71 @@ mission "Terraforming 5"
 			`	Amy says, "It will take a year or two for things to begin to settle down. Then we can figure out what our next steps should be."`
 			`	"I just hope we haven't made things worse than they were," says Alaric. He hands you your payment of <payment>. "Anyway, thanks for your help, <first>. It sounds like there's nothing more to be done here for the time being."`
 		event "terraforming Rand"
+
+ "Building Skytower 1"
+	name "Building Skytower [1]"
+	description "Escort a convoy of freighters to <destination>, where they will pick up materials for an new skytower on Earth."
+	minor
+	source Earth
+	destination Alphard
+	to offer
+		random < 10
+                reputation republic: 1<
+                has "event: war ends: done"
+	passengers 1
+	
+	on offer
+		conversation
+			`As you are walking through the spaceport, you notice people crowding around windows. You walk to one of the less crowded windows and notice a tall black tower jutting out of the ground. This is no small feat of engineering, as even in it's incomplete state the building still dwarfes the others around it. It is quite apparently skytower`
+                        `As you are staring out the window, a man in a navy uniform walks up next to you. "Magnificent, isn't it?" He asks.`
+			choice
+				`	"Yes, indeed."`
+				`	"Is this a skytower?"`
+				
+			`	"It is," he says. "Now, I know what you're thinking. 'Wern't skytowers prohibited when the one in 2235 fell?' They were. But, with the ever growing population, this planet is turning into an urban world. On top of that, the greenhouse gas emmisions from the billions of people that call this world home are destroying the atmosphere, and the oceans are rising steadily. We need to get lots of people off the planet, and fast too if we want to save our homeworld from becoming a ecological disaster."`
+			choice
+				`	"This is scary stuff. Is there any way I can help?"`
+                                `       "Wow. What can we do?"`
+
+                       `        "There is somthing you can do. We need some cargo ships escorted to Alphard to pick up some super-strong nanotube buckyballs that hopefully should help strengthen this." He gestures toward the tower. "I also need you to bring some scientists back as well. Are you up to the challange?"`
+                        choice
+                                `       "Yes I am"`
+				`       "Sorry, but I have somthing urgent to do"` 
+                                decline	
+			`       "Thanks!" He says, "By the way, I'm admrial Michel Farford." "<first> <last>." you say. You shake hands, and you show Michel to his bunk."`
+	
+	on complete
+		payment 200000
+ 
+               #Fleet of freighter ships is needed here before merge
```

Comment:
Try 3-5 Bulk freighters. You may want to put in some Pirate fleets in both missions to cause trouble. You will also need a `to fail` condition.